### DISCUSSION TITLE
Probando Discussion conectada a correo electrónico

### DISCUSSION THREAD

**Discussion by TheMoys** [TheMoys/24-25-IdSw1-SDR#1] [Q&A] [answered]
Esto es una prueba @TheMoys

**Comment by TheMoys** [comment]
Probando again @TheMoys

**Comment by TheMoys** [comment]
@diegojiimenez Contesta

**Comment by diegojiimenez** [comment]
Contestando desde el correo

**Comment by TheMoys** [comment]
Contestando a la contestación desde el correo...

**Comment by TheMoys** [comment]
Probando borrar texto innecesario...

**Comment by diegojiimenez** [comment, marked as answer]
Si se borro creo solo queda lo de este mensaje

**Comment by TheMoys** [comment]
@diegojiimenez 
[dieta-11.pdf](https://github.com/user-attachments/files/18180815/dieta-11.pdf)
Envio de pdf por correo
